### ISSUE TITLE
Unable to Get same results in the CRVQ paper

### ISSUE THREAD

**Issue by Dian-ZL** [xuyuzhuang11/CRVQ#1]
I used CRVQ on LLaMA2 7B, but there is a significant difference between the results on Wiki2 and C4 compared to the original results.

<img width="486" height="114" alt="Image" src="https://github.com/user-attachments/assets/47dec09b-c9ec-4b97-ad59-37f5a5742b73" />

The parameters are as follows:
`python main.py 
    $MODEL_PATH 
    $DATASET_PATH 
    --nsamples=2048 
    --val_size=256 
    --model_seqlen=4096 
    --num_codebooks=4 
    --nbits_per_codebook 8 8 8 8 
    --multibook_ratio=0.02 
    --shuffle_rule=3 
    --out_group_size=1 
    --in_group_size=8 
    --beam_size=1 
    --relative_mse_tolerance=0.01 
    --max_epochs=100 
    --finetune_lr=2.12e-5 
    --finetune_adam_beta1=0.90 
    --finetune_adam_beta2=0.95 
    --finetune_keep_best 
    --finetune_batch_size=64 
    --local_batch_size=16 
    --finetune_max_epochs=25 
    --finetune_early_stop=3 
    --offload_activations 
    --save $SAVE_PATH `

Did I set the parameter values incorrectly? By the way, I use **wikitext2** data for fine-tuning.

**Comment by xuyuzhuang11** [comment]
Hi~ Thanks for your attention! I think your reproduction do not run end-to-end fine-tuning. Please run fine-tune.sh in our root directory and then evaluate it. Moreover, I recommend to use RedPajama to reproduce the results in our paper. : )

**Comment by Dian-ZL** [comment]
THX for the reply! I will try again according to your suggestions.

**Comment by xuyuzhuang11** [comment]
Hi, friend~Would you have other questions on this issue? If not, I will close this issue. :-)
